modules/serverless-tech-preview-features.adoc

```diff
@@ -13,54 +13,63 @@ The following table provides information about which {ServerlessProductName} fea
 // OCP + OSD table
 ifdef::openshift-enterprise,openshift-dedicated[]
 .Generally Available and Technology Preview features tracker
-[cols="3,1,1,1",options="header"]
+[cols="4,1,1,1",options="header"]
```

Comment:
This change probably breaks the table, see screenshot below:
![image](https://user-images.githubusercontent.com/336823/205853253-8706b941-f584-4fd3-a109-3b9808dbc868.png)

Reply:
Perhaps it should be `[cols="3,1,1,1,1",options="header"]`

Reply:
It was broken before the change and I suggested that increasing it but it is still breaks :sweat: 
https://github.com/openshift/openshift-docs/pull/53365#discussion_r1037827738

Reply:
Opps. I'm late to this party :)